assets/js/blocks/product-collection/editor.scss

```diff
@@ -24,3 +24,13 @@
 		}
 	}
 }
+
+// Price Range Filter
+.wc-block-product-price-range-control {
+	max-width: 200px;
```

Comment:
What's the purpose of this limit? I think we could let the controls occupy the full Inspector Controls width as it stands out now from other filters:
<img width="290" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/618b8ab5-c340-4d6d-bf67-3c74ea7172b2">